### ISSUE TITLE
dayOfYear method missing in typescript

### ISSUE THREAD

**Issue by ansf** [spencermountain/spacetime#333]
Using version 7.1.2, the following code does not compile in typescript:

```
import spacetime from 'spacetime'
const today = spacetime().dayOfYear()
```

error is:

`error TS2339: Property 'dayOfYear' does not exist on type 'Spacetime'.`

According to https://observablehq.com/@spencermountain/spacetime-api, this method should exist.

Am I doing something wrong, or this is method missing in the types?

**Comment by spencermountain** [comment]
good catch!
yep, missing from the types. fixed on dev, will add this to the next release.
thanks

**Comment by spencermountain** [comment]
released as 7.1.3
cheers
